<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QVTKTransformation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QOpenGLWidget" name="openGL"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Scene"/>
    <addaction name="actionClose_Scene"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Data"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QMenu" name="menuAdd_Data">
     <property name="title">
      <string>Add Data</string>
     </property>
     <addaction name="actionSphere"/>
     <addaction name="actionArrow"/>
     <addaction name="actionCone"/>
     <addaction name="actionCube"/>
     <addaction name="actionCylinder"/>
     <addaction name="actionDisk"/>
    </widget>
    <addaction name="menuAdd_Data"/>
    <addaction name="actionRemove_Active_Data"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Scene">
   <property name="text">
    <string>New Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose_Scene">
   <property name="text">
    <string>Close Scene</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRemove_Active_Data">
   <property name="text">
    <string>Remove Active Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="text">
    <string>Arrow</string>
   </property>
  </action>
  <action name="actionCone">
   <property name="text">
    <string>Cone</string>
   </property>
  </action>
  <action name="actionCube">
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="actionCylinder">
   <property name="text">
    <string>Cylinder</string>
   </property>
  </action>
  <action name="actionDisk">
   <property name="text">
    <string>Disk</string>
   </property>
  </action>
  <action name="actionImport_Data">
   <property name="text">
    <string>Import Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
